microsoft-edge/webview2/samples/wv2deploymentwixcustomactionsample.md

```diff
@@ -138,7 +138,7 @@ If you want the app to download the Evergreen WebView2 Runtime Bootstrapper (`Mi
 
 1. Comment out other `<Binary>` and `<CustomAction>` elements under `Step 4`.
 
-1. Under `<!-- Step 5: Config execute sequence of custom action -->`, uncomment the `<Custom Action='DownloadAndInvokeBootstrapper' ...>` element below `<!-- [Download Bootstrapper] ...-->`.
+1. Under `<!-- Step 5: Config run sequence of custom action -->`, uncomment the `<Custom Action='DownloadAndInvokeBootstrapper' ...>` element below `<!-- [Download Bootstrapper] ...-->`.
```

Comment:
```suggestion
1. Under `<!-- Step 5: Config execute sequence of custom action -->`, uncomment the `<Custom Action='DownloadAndInvokeBootstrapper' ...>` element below `<!-- [Download Bootstrapper] ...-->`.
```
reverting b/c code
https://github.com/MicrosoftEdge/WebView2Samples/blob/main/SampleApps/WV2DeploymentWiXCustomActionSample/Product.wxs - line 64 & 65; 65 is <InstallExecuteSequence>

An xml comment in the code repo is out of scope; would need to create a PR in that code repo before (or simult. w/) changing here in docs repo. Such a PR might get rejected, since the xml code itself uses the word/phrase 'ExecuteSequence': line 65 is <InstallExecuteSequence>.

Style guide doesn't here discuss Dev coding docs:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/execute
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/run-vs-execute